### PR TITLE
3.1.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-02-19)
+
+- Added blur event to select field component. [#11](https://github.com/blackbaud/skyux-select-field/pull/11)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: I'd put "blur" in back ticks: `blur`.

File: package.json [hunk 2]
```diff
@@ -37,8 +37,8 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@blackbaud/skyux": "2.38.0",
+    "@blackbaud/skyux": "2.40.0",
     "@blackbaud/skyux-builder": "1.30.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: Since we're updating packages, Builder is up to `1.31.1` now.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: done

File: package.json (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: done

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**